libpldmresponder/fru.cpp

```diff
@@ -437,8 +438,36 @@ void FruImpl::removeIndividualFRU(const std::string& fruObjPath)
      }*/
     sendPDRRepositoryChgEventbyPDRHandles(
         std::move(std::vector<ChangeEntry>(1, deleteRecordHdl)),
-        std::move(std::vector<uint8_t>(1, PLDM_RECORDS_DELETED))); // need to
-    //  send both remote and local records. Phyp keeps track of bmc only records
+        std::move(std::vector<uint8_t>(1, PLDM_RECORDS_DELETED)));
+
+    std::vector<uint16_t> effecterIDs = findEffecterIds(
+        pdrRepo, 0 /*tid*/, removeEntity.entity_type,
+        removeEntity.entity_instance_num, removeEntity.entity_container_id);
+
+    for (auto& ids : effecterIDs)
+    {
+        auto delEffecterHdl = pldm_delete_by_effecter_id(pdrRepo, ids, false);
```

Comment:
check if the handle is 0 before sending the event ?

Reply:
done